dot/network/discovery.go

```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"time"
 
+	gossamermetrics "github.com/ChainSafe/gossamer/dot/metrics"
+	"github.com/ethereum/go-ethereum/metrics"
```

Comment:
```suggestion
	"github.com/ChainSafe/gossamer/dot/metrics"
	ethmetrics "github.com/ethereum/go-ethereum/metrics"
```
we should probably reserve the generic one for our metrics package.